packages/compass-preferences-model/src/feature-flags.ts

```diff
@@ -7,57 +14,47 @@ export type FeatureFlagDefinition = {
    * - 'released': the feature flag is always enabled, is not read from disk and cannot be disabled from settings.
    */
   stage: 'development' | 'preview' | 'released';
+  /**
+   * Defines the feature flag name to be picked up from Atlas Cloud. If set to a
+   * non-null value and Atlas Cloud feature flag exists and was passed to
+   * compass-web, it will take precedence over the "release stage"
+   * configuration, otherwise the `stage` logic will apply.
+   */
+  atlasCloudFeatureFlagName: string | null;
```

Comment:
Explicit `null` for backwards compatibility and because we don't always need a feature flag on the mms side